validation/linux_uid_mappings.go

```diff
@@ -1,14 +1,38 @@
 package main
 
 import (
+	"os"
+	"path/filepath"
+
 	"github.com/opencontainers/runtime-tools/validation/util"
 )
 
+func mkdir(path, subdir string) error {
+	pathName := filepath.Join(path, subdir)
+	err := os.MkdirAll(pathName, 0700)
+	if err != nil {
+		return err
+	}
+	err = os.Chown(pathName, 1000, 1000)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
 func main() {
 	g := util.GetDefaultGenerator()
 	g.AddOrReplaceLinuxNamespace("user", "")
-	g.AddLinuxUIDMapping(uint32(1000), uint32(0), uint32(3200))
-	err := util.RuntimeInsideValidate(g, nil)
+	g.AddLinuxUIDMapping(uint32(1000), uint32(0), uint32(2000))
+	g.AddLinuxGIDMapping(uint32(1000), uint32(0), uint32(3000))
+	err := util.RuntimeInsideValidate(g, func(path string) error {
+		_ = mkdir(path, "proc")
+		_ = mkdir(path, "dev")
+		_ = mkdir(path, "sys")
```

Comment:
If we need these (and it sounds like we do for now), I think they should be part of [`PrepareBundle`][1].  Until we have clearer spec wording around creation, I'd rather not rely on the details of runc's “sometimes we can do this” implementation.

[1]: https://github.com/opencontainers/runtime-tools/blob/v0.5.0/validation/util/test.go#L105-L122